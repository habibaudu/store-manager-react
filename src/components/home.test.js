import React from 'react';
import { shallow } from 'enzyme';
import Home from './home';

describe('HomePage Component', () => {
  it('should render the Landing Page', () => {
    const component = shallow(<Home />);
    expect(component.exists()).toBe(true);
    expect(component).toMatchSnapshot();
  });
});
