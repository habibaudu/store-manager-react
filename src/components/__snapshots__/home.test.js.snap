// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HomePage Component should render the Landing Page 1`] = `
<Fragment>
  <Main />
  <Footer />
</Fragment>
`;
